pkg/localmetrics/localmetrics.go

```diff
@@ -56,6 +60,10 @@ var (
 		Help:        "The duration it takes to reconcile a ClusterDeployment",
 		ConstLabels: prometheus.Labels{"name": "certman-operator"},
 	})
+	MetricCertRequestsCount = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
Does this have to be a `Vec` type?

Reply:
The only interest (on top of the alignment with other gauges) is to ease future addition of labels, but it does not need to. I can update it with ConstLabel for the operator name indeed. 
What do we prefer ?

Reply:
Let's keep it a simple gauge for now.